frontend/packages/dev-console/src/components/pipelines/resource-overview/DynamicResourceLinkList.tsx

```diff
@@ -13,21 +14,27 @@ export type ResourceModelLink = {
 type DynamicResourceLinkListProps = {
   links: ResourceModelLink[];
   namespace: string;
-  title: string;
+  title?: string;
+  addSpaceBelow?: boolean;
 };
 
 const DynamicResourceLinkList: React.FC<DynamicResourceLinkListProps> = ({
   links = [],
   namespace,
   title,
+  addSpaceBelow = true,
```

Comment:
A suggestion instead of having to place a default would be to make this a "removal" command. Easier to use as a react component prop & you don't need to place a default as it's a boolean. Could go with `removeSpacing`... this is all a suggestion, I am good with both ways.